patches/node/chore_add_ability_to_prevent_warn_non_context-aware_native_modules.patch

```diff
@@ -46,17 +46,18 @@ index c51a892e1bcd3df6ea8af952c8200975c7f4ac7e..9a4a7780f4ac1bcc0855cc9e3eed904d
  #include "util.h"
  
  #if HAVE_OPENSSL
-@@ -463,6 +464,13 @@ void DLOpen(const FunctionCallbackInfo<Value>& args) {
-     }
+@@ -466,10 +466,12 @@ void DLOpen(const FunctionCallbackInfo<Value>& args) {
  
      if (mp != nullptr) {
-+      if (mp->nm_context_register_func == nullptr) {
+       if (mp->nm_context_register_func == nullptr) {
+-        if (env->options()->force_context_aware) {
 +        if (env->force_context_aware()) {
```

Comment:
how come we're moving this from `env->options()` to `env`?

Reply:
this is becuase i upstreamed this patch partially but i felt it was easier to retain our existing approach until we remove the warning aspect. To access the new functionality we'd need to pass `--force-context-aware` as an env flag and i thought for now it might almost be more confusing to mix the two approaches while we're still warning folks.

Open to other opinions tho!

Reply:
i'll add a comment to the patch indicating when we should take advantage of upstream flag tho, ty for the reminder :)